dotaservice/protos/DotaService.proto

```diff
@@ -25,8 +25,21 @@ enum HostMode {
   HOST_MODE_GUI_MENU = 2;
 }
 
+enum SelectionType {
+  SELECTION_TYPE_PICK = 0;
+  SELECTION_TYPE_BAN = 1;
+  // For Ability Draft
+  SELECTION_TYPE_ABILITY_PICK = 2;
+}
+
 message Empty {}
 
+message HeroSelection {
```

Comment:
I think this is really great to get us started - how do you envison the information of the opposite pick to be returned?

Reply:
Also, how do we specify how we set up the game? How do we specify if we want to control the bots, or if we want to use default bots?